src/StaticFAQ/FAQCategoryList.js

```diff
@@ -182,6 +182,12 @@ class RawFAQCategoryList extends React.Component<Props> {
       return <StaticFAQError />;
     }
 
+    const FAQCategory = idx(rendererProps.props, _ => _.FAQCategory);
```

Comment:
you need to do it inside `rendererProps.props` condition, this would break Loader below

Reply:
fixed